### PR TITLE
WIP: pull in changes to chain names from superchain

### PR DESCRIPTION
**Pull request by geoknee**
See https://github.com/ethereum-optimism/superchain-registry/pull/153

### CODE REVIEW

File: op-chain-ops/cmd/op-upgrade/main.go [hunk 1]
```diff
@@ -228,23 +228,17 @@ func entrypoint(ctx *cli.Context) error {
 // with a canonical naming scheme. If an empty string is returned, then
 // it means that the chain is not supported yet.
 func toDeployConfigName(cfg *superchain.ChainConfig) (string, error) {
```
Comment (geoknee) [inline op-chain-ops/cmd/op-upgrade/main.go:230]: We might be able to just use the chain's short name as the deploy config filename.

Comment (tynes) [reply op-chain-ops/cmd/op-upgrade/main.go:230]: It would def be ideal to remove this function by canonicalizing the names

Comment (geoknee) [reply op-chain-ops/cmd/op-upgrade/main.go:230]: I'm planning on exposing each chain's "short name" from the registry, which we will enforce to match the short name registered with https://chainid.network/. 

What do we think of using this as the canonical name throughout our stack? It is required to be unique according to the maintainers of that site https://github.com/ethereum-lists/chains.

We could also use the long name.

 Some of the short names might not be what we would choose. Example from chainid.network: 

```json
{
    "name": "OP Mainnet",
    "chainId": 10,
    "shortName": "oeth",
    "networkId": 10,
...
}
```

Comment (geoknee) [reply op-chain-ops/cmd/op-upgrade/main.go:230]: (Motivation to use these existing names is to reduce duplication)

**Pull request closed without merging**